src/Combat.cpp

```diff
@@ -86,9 +87,20 @@ void SpellMovementSystem::on_tick()
 }
 
 void SpellCollisionSystem::process_signal(SpellPlayerCollisionSignal& signal){
-    if (is_ok(signal.player))
-        despawn(signal.player);
-    
+    if (is_ok(signal.player) && is_ok(signal.spell))
+    {
+        if (auto* spell = MutAccessComponentById<Spell>::get(signal.spell))
+		{
+			if (auto* status = MutAccessComponentById<Status>::get(signal.player))
+            {
+                status->health -= spell->damage;
+                if(status->health <= 0) 
+                {
+                    despawn(signal.player);
+                }
+            }
+		}
+    }
```

Comment:
There seem to be some alignment issues with these brackets.
Looks like tabs and spaces are used interchangeably while tabs should be primary.

Reply:
These issues are only present in the pull request's changed files for some reason. The indentation seems fine in the original source code, having used only tabs and no spaces.

Reply:
No, it's also present in the code now.
`src/Combat.cpp` mostly uses spaces and tabs are used on the parts that are misaligned.
Could be that your editor is configured differently and it's showing it as aligned.